### PR TITLE
Removed Beta tag from handle, and readMany APIs

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Removed Beta tag from handle, and readMany APIs

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java [hunk 1]
```diff
@@ -79,7 +79,6 @@ public Duration getDuration() {
      * Regions contacted for this request
      * @return set of regions contacted for this request
      */
-    @Beta(value = Beta.SinceVersion.V4_9_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public Set<URI> getRegionsContacted() {
```
Comment (moderakh) [inline sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java:82]: can we postpone GAing the `getRegionsContacted()` API for now.
I suspect we may need to return a different type, e.g., Set<URL> etc.

Comment (mbhaskar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java:82]: Also, the feedResponseDiagnostics doesnt return all the values for this and currently just null. So its worth waiting to fix that part too for GA

**Review by moderakh — APPROVED** [review]
I am signing off pending one comment on the `getRegionsContacted()`. 

"can we postpone GAing the getRegionsContacted() API for now.
I suspect we may need to return a different type. perhaps a `Set<URL> `or `Set<String>`"

File: sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java]: @moderakh  - makes sense, reverted this API back to Beta.

**Pull request merged**